docs/design/architecture.md

```
@@ -190,8 +192,8 @@ stack is now lighter-weight.
 database is now contained within each node, overall cross-node and cross-cluster
 (HostedControlPlane, ManagedSaaS) chatter is decreased and traffic security can be increased.
 
-## Default Mode versus Interconnect Mode
+## Central Mode versus Interconnect Mode
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)